patches/benchmarks.R-cran-rbenchmark.1.0.0.diff

```diff
@@ -0,0 +1,59 @@
+diff --git a/benchmarks/Makefile b/benchmarks/Makefile
+index 645d42b0ec2c..3bd53ba2ba3b 100644
+--- a/benchmarks/Makefile
++++ b/benchmarks/Makefile
+@@ -3,6 +3,7 @@
+ 
+     COMMENT = Benchmarking tools
+ 
++    SUBDIR += R-cran-rbenchmark
+     SUBDIR += ali
+     SUBDIR += apib
+     SUBDIR += autobench
+diff --git a/benchmarks/R-cran-rbenchmark/Makefile b/benchmarks/R-cran-rbenchmark/Makefile
+new file mode 100644
+index 000000000000..b3d6890b5401
+--- /dev/null
++++ b/benchmarks/R-cran-rbenchmark/Makefile
+@@ -0,0 +1,16 @@
++# $FreeBSD$
++# Created by: Guangyuan Yang <ygy@FreeBSD.org>
```

Comment:
I found that those two lines are usually in reverse order, but this is definitely not important :-)
(especially we will drop `$FreeBSD$` when switching to git.

Reply:
Oh nice find! Will switch all these in this and all other patches.